www/components/shelf/shelf.js

```diff
@@ -120,11 +121,11 @@ class shelf extends LitElement {
 
       return listItems;
     };
-    /* eslint-enable */
 
+    /* eslint-enable */
```

Comment:
Do we actually need this `/* eslint */`?  What was it disabling?  I think I should make another issue because we might have a few of these we don't need anymore.

Plus, I would prefer we only enable it per rule, if we do need an exception.  Like I said, maybe another issue, but curious since it's showing in the diff.

Reply:
Yes it's required, you'll note the section:

```
        listItems = html`
          <ul>
            ${list.items.map(item => {
              return html`
                <li id="index_${item.index}" class="${list.selected ? '' : 'hidden'}"><a @click=${()=> this.goTo(`#${item.id}`)}">${item.name}</a></li>
              `;
            })}
          </ul>
        `;
```

Requires it to be disabled because of indentation.  Then below that function we re-enable eslint

Reply:
AH, right, that thing...